virtdata-lib-basics/src/main/java/io/nosqlbench/virtdata/library/basics/shared/from_long/to_long/HostHash.java

```diff
@@ -60,10 +60,10 @@ private static long computeHostHash() {
                 distinctNames.add(iface.getHostName());
             });
             List<String> nameList = new ArrayList<>(distinctNames);
-            nameList.sort(String::compareTo);
+            Collections.sort(nameList);
             Murmur3F m3f = new Murmur3F(0);
             m3f.reset();
-            distinctNames.forEach(
+            nameList.forEach(
```

Comment:
i assume this was the idea behind having a sorted list i.e. more determinism?
otherwise `nameList` was unused.

Reply:
Correct, corrected!